chart/README.md

```diff
@@ -541,14 +541,64 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
```

Comment:
We need to add something similar to **NEW FROM VERSION** to indicate the chart version this feature first enabled. 
Perhaps 
```
@Since version 166.0.0
```

Reply:
Commit: 095d334652f4c759547391f8d3d0f744be7cf230